internal/config/e2eConfig.go

```diff
@@ -53,6 +52,10 @@ type Run struct {
 	Waits   []Wait `yaml:"wait"`
 }
 
+func (r Run) GetRunCommand() string {
+	return r.Command
+}
```

Comment:
If we don't need extra operations when getting a public field, let's use `Run.Command` directly